models/netsuite2.yml

```diff
@@ -55,6 +55,7 @@ models:
             - transaction_id
             - accounting_period_id
             - account_name
+            - account_id
```

Comment:
I'd like to at least add account_id to the unique test here. Reason being that account_name may not necessarily be unique across accounts so it's not enough to test uniqueness, while we can be sure that `account_id` ID unique. (and we test uniquness on account_id in stg_netsuite2__accounts, not account_name anyways)

Reply:
But we may need to add more grains to this test depending on what we find out from the customer who's seeing this fail, so this may change